account.go

```diff
@@ -61,37 +61,54 @@ const (
-	DebitNegativeBal, NoDebitNegativeBal bool
-	TOSAcceptance                        *TOSAcceptanceParams
-	Type                                 AccountType
+	Params               `form:"*"`
+	Country              string                        `form:"country"`
+	Email                string                        `form:"email"`
+	DefaultCurrency      string                        `form:"default_currency"`
+	Statement            string                        `form:"statement_descriptor"`
+	BusinessName         string                        `form:"business_name"`
+	BusinessUrl          string                        `form:"business_url"`
+	BusinessPrimaryColor string                        `form:"business_primary_color"`
+	SupportPhone         string                        `form:"support_phone"`
+	SupportEmail         string                        `form:"support_email"`
+	SupportUrl           string                        `form:"support_url"`
+	FromRecipient        string                        `form:"from_recipient"`
+	PayoutStatement      string                        `form:"payout_statement_descriptor"`
+	ExternalAccount      *AccountExternalAccountParams `form:"external_account"`
+	LegalEntity          *LegalEntity                  `form:"legal_entity"`
+	PayoutSchedule       *PayoutScheduleParams         `form:"payout_schedule"`
+	DebitNegativeBal     bool                          `form:"debit_negative_balances"`
+	NoDebitNegativeBal   bool                          `form:"debit_negative_balances,invert"`
+	TOSAcceptance        *TOSAcceptanceParams          `form:"tos_acceptance"`
+	Type                 AccountType                   `form:"type"`
 }
 
 // AccountListParams are the parameters allowed during account listing.
 type AccountListParams struct {
-	ListParams
+	ListParams `form:"*"`
 }
 
 // AccountExternalAccountParams are the parameters allowed to reference an
 // external account when creating an account. It should either have Token set
 // or everything else.
 type AccountExternalAccountParams struct {
-	Params
-	Account, Country, Currency, Routing, Token string
+	Params            `form:"*"`
+	Account           string `form:"account_number"`
+	AccountHolderName string `form:"account_holder_name"`
+	AccountHolderType string `form:"account_holder_type"`
+	Country           string `form:"country"`
+	Currency          string `form:"currency"`
+	Routing           string `form:"routing_number"`
+	Token             string `form:"token"`
 }
 
 // PayoutScheduleParams are the parameters allowed for payout schedules.
 type PayoutScheduleParams struct {
-	Delay, MonthAnchor uint64
-	WeekAnchor         string
-	Interval           Interval
-	MinimumDelay       bool
+	Delay        uint64   `form:"delay_days"`
+	MonthAnchor  uint64   `form:"monthly_anchor"`
+	WeekAnchor   string   `form:"weekly_anchor"`
+	Interval     Interval `form:"interval"`
+	MinimumDelay bool     `form:"delay_days_minimum"`
```

Comment:
I'm a bit confused by this one. `delay_days_minimum` does not exist in the API. This boolean decides if we should explicitly pass `delay_days: "minimum"` to the API instead of a number.
The code you removed did this
```
} else if t.MinimumDelay {
  values.Add("payout_schedule[delay_days]", "minimum")
}
```

Reply:
Ah, great point. I misunderstood that one. Added custom encoder and test in 36451a8.